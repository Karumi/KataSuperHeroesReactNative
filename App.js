import KataSuperHeroesReactNative from "./src/App";

export default KataSuperHeroesReactNative;
