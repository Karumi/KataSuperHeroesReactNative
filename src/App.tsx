import * as React from "react";
import { Text } from "react-native";

class App extends React.Component {
    public render() {
        return <Text>Hello Super Heroes</Text>;
    }
}

export default App;
