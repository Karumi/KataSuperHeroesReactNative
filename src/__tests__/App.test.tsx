import * as React from "react";
import * as renderer from "react-test-renderer";

import App from "../App";

it("renders correctly with the default configuration", () => {
    const component = renderer
        .create(<App />)
        .toJSON();
    expect(component).toMatchSnapshot();
});
