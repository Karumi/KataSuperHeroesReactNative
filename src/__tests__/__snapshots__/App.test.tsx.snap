// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with the default configuration 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  Hello Super Heroes
</Text>
`;
